.github/problem-matchers/flutter-analyze.json

```diff
@@ -0,0 +1,32 @@
+{
+  "problemMatcher": [
+    {
+      "owner": "flutter-analyze",
+      "pattern": [
+        {
+          "regexp": "^\\s*(error|warning) • (.*?) • (.*):([0-9]+):([0-9]+) • (.*)$",
+          "severity": 1,
+          "message": 2,
+          "file": 3,
+          "line": 4,
+          "column": 5,
+          "code": 6
+        }
+      ]
+    },
+    {
+      "owner": "flutter-analyze-info",
+      "severity": "notice",
+      "pattern": [
+        {
+          "regexp": "^\\s*info • (.*?) • (.*):([0-9]+):([0-9]+) • (.*)$",
```

Comment:
Maybe we should make it failing, and instead use ignores inside the code to suppress, if we feel we need to fix it later

Reply:
It's much easier to forget suppressed warnings like that. We do want to fix them all, they are 2 actual valid warnings plus a bunch of deprecation notices, which will eventually turn into errors.

If you are concerned about the annotations added to all future PRs, then maybe we should wait until all of these warnings have been fixed before merging this.